geode-core/src/main/java/org/apache/geode/management/internal/cli/shell/JmxOperationInvoker.java

```diff
@@ -407,7 +374,7 @@ public static String checkAndConvertToCompatibleIPv6Syntax(String hostAddress) {
 
   @Override
   public void handleNotification(Notification notification, Object handback) {
-    if (JMXConnectionNotification.class.isInstance(notification)) {
+    if (notification instanceof JMXConnectionNotification) {
```

Comment:
Just curious why you prefer `notification instanceof JMXConnectionNotification` over the newer syntax of `JMXConnectionNotification.class.isInstance(notification)`.

Reply:
No particular reason at all... I guess old habits die hard, and the fact that `IntelliJ` flags `JMXConnectionNotification.class.isInstance` as a warning made me finally decide to use the old way.